packages/frameworks/src/frameworks.ts

```diff
@@ -554,15 +554,15 @@ export const frameworks = [
     logo: 'https://api-frameworks.vercel.sh/framework-logos/solid.svg',
     tagline: 'Simple and performant reactivity for building user interfaces.',
     description: 'A Solid app, created with SolidStart.',
-    website: 'https://solidjs.com',
+    website: 'https://start.solidjs.com',
     envPrefix: 'VITE_',
     detectors: {
       every: [
         {
           matchPackage: 'solid-js',
         },
         {
-          matchPackage: 'solid-start',
+          matchPackage: '@solidjs/start',
         },
       ],
```

Comment:
So this would break framework detection for older versions of SolidStart. How about this instead?

```suggestion
      some: [
        {
          matchPackage: 'solid-start',
        },
        {
          matchPackage: '@solidjs/start',
        },
      ],
```

Reply:
We may want multiple entries. Some projects will not have `vinxi`.